integration_tests/e2e/overview/preInductionOverview.cy.ts

```diff
@@ -1,36 +1,20 @@
 import Page from '../../pages/page'
 import OverviewPage from '../../pages/overview/OverviewPage'
-import CreateCiagInductionPage from '../../pages/ciagUi/createCiagInductionPage'
 import Error500Page from '../../pages/error500'
 import CreateGoalsPage from '../../pages/goal/CreateGoalsPage'
+import HopingToWorkOnReleasePage from '../../pages/induction/HopingToWorkOnReleasePage'
 
 context('Prisoner Overview page - Pre Induction', () => {
+  const prisonNumber = 'G6115VJ'
+
   beforeEach(() => {
-    cy.task('reset')
-    cy.task('stubSignInAsUserWithEditAuthority')
-    cy.task('stubAuthUser')
-    cy.task('stubGetHeaderComponent')
-    cy.task('stubGetFooterComponent')
-    cy.task('stubPrisonerList')
-    cy.task('stubCiagInductionList')
-    cy.task('stubActionPlansList')
-    cy.task('getPrisonerById', 'G6115VJ')
-    cy.task('getPrisonerById', 'A00001A')
+    cy.signInAsUserWithEditAuthorityToArriveOnPrisonerListPage()
```

Comment:
Simplified the beforeEach hook by using the new `signInAsUserWithEditAuthorityToArriveOnPrisonerListPage` cypress command that I added the other day (the new command contains most of tasks that we were individually setting up beforehand)